ossec_security/README.md

```diff
@@ -0,0 +1,204 @@
+# Agent Integration: ossec-security
+
+## Overview
+
+[OSSEC][4] is an Open Source Host based Intrusion Detection System. It performs log analysis, integrity checking, Windows registry monitoring, rootkit detection, real-time alerting and active response. It helps to monitor and manage security events across various IT infrastructures.
+
+This integration ingests the following types of logs:
+- FTPD
+- Firewall
+- System
+- Syslog
+- SSHD
+- Pam
+- Windows
+- Web Access
+
+Visualize detailed insights into Firewall, FTPD, System, Syslog, SSHD, Pam, Windows, Web Access logs through the out-of-the-box dashboards.
+
+## Setup
+
+### Installation
+
+To install the OSSEC Security integration, run the following Agent installation command and the steps below. For more information, see the [Integration Management][6] documentation.
+
+**Note**: This step is not necessary for Agent version >= 7.54.0.
+
+Linux command
+  ```shell
+  sudo -u dd-agent -- datadog-agent integration install datadog-ossec_security==1.0.0
+  ```
+
+### Configuration
+
+#### Logs Collection
+
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in datadog.yaml:
+
+    ```shell
+    logs_enabled: true
+    ```
+2. Add this configuration block to your `ossec_security.d/conf.yaml` file to start collecting your logs.
+
+    We will be using the UDP method to collect the OSSEC alerts data.
+    See the [sample ossec_security.d/conf.yaml][8] for available configuration options.
+
+    ```yaml
+      logs:
+      - type: udp
+        port: <PORT>
+        source: ossec-security
+        service: ossec-security
+    ```
+    **Note**: It is recommended not to change the service and source values as these parameters are integral to the pipeline's operation.
+
+3. [Restart the Agent][3].
+
+#### Configure Syslog Message Forwarding from OSSEC.
```

Comment:
```suggestion
#### Configure syslog message forwarding from OSSEC
```

Reply:
Unless "Syslog Message Forwarding" is an official feature name, in which case leave it capitalized.

Reply:
Updated as per suggestion